README.rst

```diff
@@ -88,6 +88,9 @@ Available Features
   mobile signal (LTE/UMTS/GSM `signal strength <#mobile-signal-strength>`_,
   `signal quality <#mobile-signal-quality>`_,
   `access technology in use <#mobile-access-technology-in-use>`_)
+* Maintains a record of `WiFi sessions <#monitoring-wifi-sessions>`_ with clients'
+  MAC address and vendor, session start and stop time and connected device
+  along with other information.
```

Comment:
The other points don't have a final dot so let's maintain consistency.

Let's make clear that both currently open sessions and past sessions can be viewed and filtered.